machine/meta-xt-images-rcar-gen3/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_git.bbappend

```diff
@@ -1,2 +1,3 @@
 # Start cores in EL2 mode
 ATFW_OPT_append = ' RCAR_BL33_EXECUTION_EL=BL33_EL2'
+ADDITIONAL_ATFW_OPT_append = ' RCAR_BL33_EXECUTION_EL=BL33_EL2'
```

Comment:
The proper fix would be to assign this value once with full understanding of what happens to ATFW_OPT variable.
The way it is now is kind of hacky and might really mask the real issue,  so the current behavior needs to be properly explained.